lib/salus/processor.rb

```diff
@@ -7,27 +7,42 @@ class InvalidConfigSourceError < StandardError; end
 
     attr_reader :config, :report
 
+    # If configuration sources are not provided, we'll automatically scan the
+    # repo root for a configuration file with this default name
     DEFAULT_CONFIG_SOURCE = "file:///salus.yaml".freeze
 
     def initialize(configuration_sources = [], repo_path: DEFAULT_REPO_PATH)
       @repo_path = repo_path
 
       # Add default file path to the configs if empty.
       configuration_sources << DEFAULT_CONFIG_SOURCE if configuration_sources.empty?
+      valid_sources = []
+      source_data = []
 
       # Import each config file in order.
-      files = configuration_sources.map { |source| fetch_config_file(source, repo_path) }
+      configuration_sources.each do |source|
+        body = fetch_config_file(source, repo_path)
+        if !body.nil?
```

Comment:
Does it make sense to output an error/warning if body is nil?

Reply:
I thought about that, but since we look for `salus.yaml` by default, if you run this on a repo without a config it will always output an error?